services/visual-studio-marketplace/visual-studio-marketplace-rating.tester.js

```diff
@@ -4,7 +4,7 @@ const Joi = require('joi')
 const t = (module.exports = require('..').createServiceTester())
 const { withRegex, isStarRating } = require('../test-validators')
 
-const isVscodeRating = withRegex(/[0-5].[0-9]{2}\/5?\s*\([0-9]*\)$/)
+const isVscodeRating = withRegex(/[0-5]\.[0-9]{1}\/5?\s*\([0-9]*\)$/)
```

Comment:
The Marketplace actually only uses 1 decimal place so I thought this (plus the format in the `render` function) would make our badges consistent